src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/PosFutureDriftRuleTest.cs

```diff
@@ -10,6 +10,9 @@ namespace Stratis.Bitcoin.Features.Consensus.Tests.Rules.CommonRules
 {
     public class PosFutureDriftRuleTest : TestPosConsensusRulesUnitTestBase
     {
+        private const int MaxFutureDriftBeforeHardFork = 128 * 60 * 60;
```

Comment:
is this right? 128 hours was it ?  wow....

Reply:
Yes it was. Crazy huh.